### ISSUE TITLE
attachFields.js/attachFieldToNode fieldValue when not using getter doesn't get value from node.fields

### ISSUE THREAD

**Issue by baerrach** [Undistraction/gatsby-plugin-node-fields#5]
In `attachFields.js/attachFieldToNode` the fieldValue is set via:
```
  var fieldValue = (0, _ramdaAdjunct.isFunction)(getter) ? getter(node, context, boundActionCreators) : node[name];
```
When no getter is supplied this pulls the value directly from the node object.

Shouldn't this be from `node.fields` instead?

**Comment by Undistraction** [comment]
@baerrach Sorry it's taken me so long to reply. 

I understand what you are saying, however the node itself might have useful information. Your suggestion is totally valid, but I'm not going to change the functionality now, as this is just a default behaviour and easily overridable.

**Comment by baerrach** [comment]
@Undistraction that's alright.

re-reading this I'm not sure what I am even asking!
I clearly haven't articulated it well enough. My apologies, I'm normally better than that.

Perhaps I erroneously expected it to default to taking values from the frontmatter instead of the node.
That's `node.fields[name]` vs `node[name]` right?

Which means you must always specify getter, as the defaults aren't useful.

**Comment by Undistraction** [comment]
@baerrach thing is that not everyone using Gatsby is using Remark (or MDX), so defaulting to `node.frontmatter` is too specific.

**Comment by baerrach** [comment]
@Undistraction I never considered that.

I just zoomed through the README again to familiarise myself with the documentation, there is nothing that mentions `attachFieldToNode`... probably because its an internal implmentation detail.

So I went looking for `attachFields`, there really isn't much documentation around that to describe how to use it.

Perhaps all that is needed is an update to the documentation to describe how `attachFields` works and where its gets it's value from. Also including a comment to indicate that values from `frontmatter` will always need a `getter` supplied.
